server/channels/app/teams/teams_test.go

```diff
@@ -31,6 +31,47 @@ func TestCreateTeam(t *testing.T) {
 	require.Error(t, err, "Should not create a new team - team already exist")
 }
 
+func TestCreateTeamWithExperimentalDefaultChannels(t *testing.T) {
+	th := Setup(t)
+	th.UpdateConfig(func(cfg *model.Config) {
+		cfg.TeamSettings.ExperimentalDefaultChannels = []string{"channel-1", "channel-2"}
+	})
+	defer th.TearDown()
+
+	id := model.NewId()
+	team := &model.Team{
+		DisplayName: "dn_" + id,
+		Name:        "name" + id,
+		Email:       "success+" + id + "@simulator.amazonses.com",
+		Type:        model.TeamOpen,
+	}
+
+	_, err := th.service.CreateTeam(team)
```

Comment:
The signature of this method has changed since you started working on this PR, so it's breaking the tests. I'll update that
```suggestion
	_, err := th.service.CreateTeam(th.Context, team)
```